web-client/src/presenter/computeds/trialSessionsHelper.ts

```diff
@@ -27,6 +29,7 @@ export const trialSessionsHelper = (get: Get): any => {
 
   return {
     additionalColumnsShown,
+    newTrialSession: permissions.CREATE_TRIAL_SESSION,
```

Comment:
Wondering if we can give a clearer name here. `newTrialSession` seems to represent if a user has permissions to start a trial session or not.

Reply:
Good call, thanks!